package.json

```diff
@@ -1,6 +1,17 @@
 {
-  "private": true,
   "name": "tupaia",
+  "private": true,
+  "description": "Mono-repo for the Tupaia project, mapping health data across the Pacific",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia.git"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
Nice, glad there is a convention for this

Reply:
Not only that, initially I manually specified our custom license name here and `yarn install` would fail! 😲